dask/array/tests/test_reductions.py

```diff
@@ -67,6 +67,8 @@ def test_reductions_1D(dtype):
     reduction_1d_test(da.nansum, a, np.nansum, x)
     with ignoring(AttributeError):
         reduction_1d_test(da.nanprod, a, np.nanprod, x)
+        reduction_1d_test(da.nancumprod, a, np.nancumprod, x)
```

Comment:
Rather, I'm guessing you mean something like I did for xarray: https://github.com/pydata/xarray/pull/812/files#diff-58edd1ff722ba75004a4318ad649d76aR254